### PR TITLE
Fixes `Transaction.ToJson()`

### LINKED ISSUE #1069: Make fee format be unified.
In ```getblocksysfee```, ```fee``` returns full decimals. However, in ```getrawtransaction```, ```fee``` has decimal point. I think it's necessary to make fee format be unified in ```RPC```, otherwise, it'll make a lot of users be comfused.
image
image

### PR DESCRIPTION
**Pull request by erikzhang**
Fixes #1069

### CODE REVIEW

**Comment by shargon** [comment]
Is related with https://github.com/neo-project/neo/issues/1069 right?

File: neo/Network/P2P/Payloads/Transaction.cs [hunk 1]
```diff
@@ -175,8 +175,8 @@ public JObject ToJson()
             json["version"] = Version;
             json["nonce"] = Nonce;
             json["sender"] = Sender.ToAddress();
-            json["sys_fee"] = new BigDecimal(SystemFee, NativeContract.GAS.Decimals).ToString();
-            json["net_fee"] = new BigDecimal(NetworkFee, NativeContract.GAS.Decimals).ToString();
+            json["sys_fee"] = SystemFee.ToString();
+            json["net_fee"] = NetworkFee.ToString();
             json["valid_until_block"] = ValidUntilBlock;
```
Comment (shargon) [inline neo/Network/P2P/Payloads/Transaction.cs]: in `FromJson` are good parsed, so obviously here is wrong

Comment (erikzhang) [reply neo/Network/P2P/Payloads/Transaction.cs]: `FromJson()` was added with the RPC SDK. So it is inconsistent. But it's fine now.

**Comment by superboyiii** [comment]
Good job. Works well.
image

**Pull request merged**